backend/src/controllers/clientCourses/finishCourse.ts

```diff
@@ -9,11 +9,13 @@ const finishCourse = async (req: Request, res: Response, next: NextFunction) =>
   try {
     const { id: clientCourseId } = req.params;
     const { status } = await getStatusProvider(clientCourseId);
-    if (status !== CourseStatus.successful) {
+    if (status !== CourseStatus.successful && status !== CourseStatus.assessment) {
       await updateClientCourseField(clientCourseId, COURSE_FILEDS.status, CourseStatus.failed);

```

Comment:
can we use status "rejected" here?
 cause on front currently we don't have status "failed" so we get "undefined"

Reply:
or you can add "failed" status to COURSE_LABELS and  to withDisabled

Reply:
Well, this is not a part of my PR, but I suppose that `rejected` is not really semantically appropriate  status. But yeah, I will add failed to Labels and withDisabled